DDF/InfDiff.hs

```diff
@@ -145,5 +144,5 @@ instance Lang r => Lang (InfDiff r) where
   state = InfDiff state
   runState = InfDiff runState
 
-diffInf :: (Vector (InfDiff r) v, DBI r) => Proxy v -> InfDiff r () x -> InfDiff r h (DiffType v x)
-diffInf p (InfDiff (Combine _ (GDiff f))) = liftEnv $ runDiff $ f p
\ No newline at end of file
+-- diffInf :: (Vector (InfDiff r) v, DBI r) => Proxy v -> InfDiff r () x -> InfDiff r h (DiffType v x)
```

Comment:
Just remove it. I know this function well enough to add it back anytime.
Turn it into an issue though.